src/pages/Donations/StatusTabs.tsx

```diff
@@ -0,0 +1,51 @@
+import { DonationsQueryParams } from "types/aws";
+
+type Props = {
+  status: DonationsQueryParams["status"];
+  changeStatus: (status: DonationsQueryParams["status"]) => void;
+};
+
+export default function StatusTabs(props: Props) {
+  const isReceived = props.status === "RECEIVED";
+
+  return (
+    <div className="flex">
+      <button
```

Comment:
I opted not to use [Headless UI Tabs](https://headlessui.com/react/tabs) as it would have forced the code to render two separate `Table` components for each status (`pending, received`), each of which would have to separately run `usePaginatedDonationRecords` for its own donations.
In addition to the above, there would added complexity of having to still pass the newly loaded donations to the parent (`index.tsx`) component so that it knows which donations to pass to `CsvExporter` button, thus duplicating the state.